org.bitcoincore.bitcoin-qt.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Name=Bitcoin Core
+Icon=org.bitcoincore.bitcoin-qt
+Comment=Connect to the Bitcoin P2P Network
+Comment[de]=Verbinde mit dem Bitcoin peer-to-peer Netzwerk
+Comment[fr]=Bitcoin, monnaie virtuelle cryptographique pair à pair
+Comment[tr]=Bitcoin, eşten eşe kriptografik sanal para birimi
+Exec=bitcoin-qt %f
```

Comment:
You could add Categories & Keywords here see https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s06.html for the specs.

Reply:
I assume we should copy the categories we are using [here](https://github.com/bitcoin-core/packaging/blob/master/debian/bitcoin-qt.desktop)? 

Currently `Categories=Office;Finance;P2P;Network;Qt;`

Reply:
Why not just using the same file for both debian & Flatpak packaging?